README.md

```diff
@@ -40,7 +40,7 @@ iOS and Android.
 ### Fast results
 
 Flutter is fast. It's powered by the same hardware-accelerated [Skia] 2D
-graphics library that underpins Chrome and Android. We architected Flutter to
+the graphics library that underpins Chrome and Android. We architected Flutter to
```

Comment:
I think the original was correct. Consider the (simplified) version: "It's powered by the same graphics library that underpins Chrome and Android". Now with modifiers to "graphics library", "powered by the same hardware-accelerated Skia 2D graphics library that underpins Chrome and Android." Alternatively, this sentence could be re-written "It's powered by the same hardware-accelerated 2D graphics library that underpins Chome and Android: [Skia]."

Reply:
Yes, I totally agree with you 👍